README.md

```diff
@@ -111,6 +111,18 @@ Remember to run these commands in the root directory of your project where the `
 Pycharm users can also use the virtual environment created by Poetry. To do this follow
 the [guides](https://www.jetbrains.com/help/pycharm/poetry.html).
 
+## Using Poe the Poet as Task Runner
+
+[Poe the Poet](https://poethepoet.natn.io/index.html) is a task runner that simplifies running common tasks in a Python project.
+
+To have the command availalbe as `poetry poe <command>` as seen in the following examples, you need to install Poe as a plugin to poetry:
+
+```bash
+poetry self add 'poethepoet[poetry_plugin]'
+```
+
+If this plugin is not installed you may have to run Poe as a script within the Poetry environment using `poetry run poe <command>`.
+
 ## Usage
```

Comment:
The `pyproject.toml` file related preparation is missing for me.

I mean: I need to prepare the `pyproject.toml` file.

1. Open the file for edit
2. Add `[tool.poe.tasks]` section
3. Add all required commends underneath this section. 

```
[tool.poe.tasks]
lint = "ruff check ."
typecheck = "mypy . --pretty"
format = "black . --check"
...
```
4. Save file